charts/greenhouse/templates/configmap-perses-dashboards.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if and .Values.plugins.enabled .Values.perses.enabled }}
+{{- range $path, $bytes := .Files.Glob "perses-dashboards/*.json" }}
```

Comment:
This condition should also be around the Plugin, right? I would opt to enable it by default and make it opt-out. Together with the other comment, it would make additional configuration in the internal config repo unnecessary.

Reply:
Yes, you are right. I have missed it. This is a great feedback.  I have fixed in f3ecefe7cee81372b4b8d25ffd67c9d1591485ea :) I have also closed the PR in the internal repo.